source/web/pages/External/App.tsx

```diff
@@ -26,8 +29,8 @@ const App: FC = () => {
             <Route path="/approveSpend" component={ApproveSpend} />
             <Route path="/">
               {!isUnlocked ?
-                <Redirect to={`/login?${location.search}`} /> :
-                <Redirect to={`/${route}?${location.search}`} />
+                <Redirect to={`/login${location.search}`} /> :
+                <Redirect to={`/${route}${location.search}`} />
```

Comment:
`location.search` always starts with a `?`. The resulting routes where ending up with multiple `?` like `??`.